### ISSUE TITLE
filetags reacts differently on two computers (converts name to lowercase)

### ISSUE THREAD

**Issue by AlpsDevABE** [novoid/filetags#28]
I have two computers with filetags installed.

On one computer filetags works as expected where if I want to add the tag "tests" then

"ABC.txt" => "ABC --test.txt"

On another computer it will force the name to lower

"ABC.txt" => "abc --test.txt"

any idea what might be causing this issue?

**Comment by novoid** [comment]
Hi,
Could you please give me more details on the computers?
At least: Operating system + version, shell environment, Python versions, filetags version

**Comment by AlpsDevABE** [comment]
Hey novoid,

Well they are both Windows 10, fully updated. Shell I jump from cmd and powershell all the time.

latest filetags version form pip

I solved the issue I was having so not sure the bug will fixable if I can no longer reproduce it.

TLDR:
My Solution was to Uninstall all things python, check cmd to make sure python is uninstalled. Reinstall python.

Details: 
I decided to remove all things python from my computer giving me issues. This is because I had discovered a few weeks ago that this computer had two python versions installed on it. I removed the two different version of PyCharm I had, removed python 3.7.2 and I could still evoke python from the shells...? Weird, it's not in the remove programs. There was a Python 3.7.0 installation which was installed but I couldn't uninstall it so I deleted that folder, checked my system paths and then reinstalled to the latest python 3.8.2 and then reinstalled your tools, and --overwrite the integrate and now everything is working as expected.

I can tell you that I did verify if it was the sendto->filetags vs the cmd prompt filetags and it was the same issue for both. 

Not sure if you still want this open as a bug since I managed to fix it. Maybe see if it is reproducable with 3.7.0 because I think my shell was taking this version over the latest version, I wanted to try to see if I could debug some of the code but I don't know how to tap into the __init__.py since readline was missing and it seems to not be a windows compatible thing.
